pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -332,6 +332,10 @@ type Controller struct {
 	//    value takes longer for user to see up-to-date node health.
 	nodeMonitorGracePeriod time.Duration
 
+	// Number of workers Controller uses to process node monitor health updates.
+	// Defaults to scheduler.UpdateWorkerSize, but can be overridden in tests.
```

Comment:
I suggest removing the "but can be overridden in tests".

While I don't think we immediately need to expose it now, I can imagine exposing it as a param of NewNodeLifecycleController in the future.